xbmc/windowing/WinSystem.h

```diff
@@ -65,6 +65,7 @@ class CWinSystemBase
   virtual bool DestroyWindow(){ return false; }
   virtual bool ResizeWindow(int newWidth, int newHeight, int newLeft, int newTop) = 0;
   virtual bool SetFullScreen(bool fullScreen, RESOLUTION_INFO& res, bool blankOtherDisplays) = 0;
+  virtual bool DPIChanged(WORD dpi, RECT windowRect) = 0;
```

Comment:
please drop this declaration it breaks build on other platforms. the declaration in WinSystemWin32.h is enough